src/app/public/modules/country-field/country-field.component.html

```diff
@@ -68,3 +68,19 @@
     {{ item.dialCode }}
   </span>
 </ng-template>
+
+<ng-template
+  #searchIconTemplateRef
+>
+  <div
+    class="sky-input-group-icon sky-input-box-icon-inset"
+  >
+    <sky-icon *skyThemeIf="'default'"
```

Comment:
Your code here has a `skyThemeIf` for `default` but the code in the component doesn't show this template at all in `deafult`. I don't think we need the `default` case here if we are never going to be showing it.

Reply:
Good point. This has been removed.